test/Bindings.test.fs

```diff
@@ -75,3 +76,75 @@ describe "expect" <| fun () ->
 
   test "should contain any" <| fun () ->
     expect.Invoke(fun () -> ()).toEqual(expect.any JS.Function)
+
+
+testList "mock timers" [
+  let withSetup f ():unit =
+    jest.useFakeTimers()
+    let timer = new System.Timers.Timer(1000.0)
+
+    f timer
+
+    jest.clearAllTimers()
+
+  yield! testFixture withSetup [
+    "should contain clearAllTimers", fun timer ->
+      let mutable h = false
+      let handler _ = h <- true
+
+      timer.Elapsed.Add handler
+      timer.Start()
+
+      jest.clearAllTimers()
+      expect.Invoke(h).toEqual(false);
+  ]
+
+  yield! testFixture withSetup [
+    "should contain runAllTimers", fun timer ->
+      let mutable h = false
+      let handler _ = h <- true
```

Comment:
This could be passed in the `withSetup`

Reply:
tried that but it didn't work for some reason, spent a fair amount of time trying to keep as much as possible within the setup routine but ended up here